Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -873,40 +872,45 @@ def add_filename_suffix(file_names: list, file_extension: str):
 
 
 def rasterize_command():  # pragma: no cover
-    url = demisto.getArg('url')
+    urls = argToList(demisto.getArg('url'))
```

Comment:
`url` can either be a list, or a single str.
This is to support url's that contain a comma.

Reply:
@jlevypaloalto, I'm not sure.
As you wrote it, if `url` contains a comma - It will be split to an array of several "urls" (in quote because the url's will be messed up,) instead of staying only one url input.

Reply:
Fixed, please review.